luna/class1-1/#4 25083.py

```diff
@@ -0,0 +1,6 @@
+print("""         ,r'"7""")
+print("""r`-_   ,'  ,/""")
+print(""" \. ". L_r'""")
+print("""   `~\/""")
+print("""      |""")
+print("""      |""")
```

Comment:
```suggestion
print(r"""         ,r'"7
r`-_   ,'  ,/
 \. ". L_r'
   `~\/
      |
      |""")
```

위와 같은 문법을 통해 한번에 출력이 가능합니다.
print(r"""<출력 값>""") 에서 r의 의미는 escape 문자를 무시한다는 뜻입니다.
해당 내용도 함께 검색해보시면 다양한 출력 방법을 공부하시는데 도움 될것 같습니다.

Reply:
몇 번 하다가 뭐가 잘 안 되길래 한 줄마다 해야하는 줄 알았어요.
제가 코드를 잘못 작성했었나봅니다!
안 그래도 aiden이랑 sean 코드 보고 알았어요.
이렇게 한 번 더 성장합니다!